src/Files.Backend/Data/Enums/AddItemDialogItemType.cs

```diff
@@ -1,13 +1,16 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-namespace Files.Backend.Enums
+namespace Files.Backend.Data.Enums
 {
 	public enum AddItemDialogItemType
 	{
 		Folder = 0,
+
 		File = 1,
+
 		Shortcut = 2,
+
 		Cancel = 3
```

Comment:
I'm not that into adding lines between enums

Reply:
This is phase 1. Preparing for adding xml comments.

Reply:
If the next phase is just comments, feel free to push another commit.

Reply:
But I have decided what I do in which PR.

> 1. Rename files with the appropriate name and put them into appropriate namespace for each.
> 2. Rename variables with preferred name to follow the guideline (The most likely to happen is that I rename property-backing-field from '_isHidden' to '_IsHidden')
> 3. Reorder variables & methods to follow the class structure guideline (Section 3 in the guildeline)
> 4. Add/Update/Improve C# XML Comments and double-slashed comments.
> 5. Refactor to improve performance. You can see what I will work on in Requirements section.

Reply:
Done. (I also did some refactoring/reordering)

Reply:
I'm ready for the next commit